src/modules/admin/components/forms/UpdateFormDefinitionPage.tsx

```diff
@@ -4,40 +4,40 @@ import FormEditor from './FormEditor';
 import Loading from '@/components/elements/Loading';
 import PageTitle from '@/components/layout/PageTitle';
 import {
-  useGetFormDefinitionForEditorQuery,
+  useGetFormIdentifierForEditorQuery,
   useUpdateFormDefinitionMutation,
 } from '@/types/gqlTypes';
 
 const UpdateFormDefinitionPage = () => {
-  const { formId } = useParams() as { formId: string };
+  const { identifier } = useParams() as {
+    identifier: string;
+  };
 
-  const { data: { formDefinition } = {}, error } =
-    useGetFormDefinitionForEditorQuery({
-      variables: { id: formId },
+  const { data: { formIdentifier } = {}, error } =
+    useGetFormIdentifierForEditorQuery({
+      variables: { identifier },
     });
 
   const [updateFormDefinition, { loading }] = useUpdateFormDefinitionMutation();
 
   if (error) throw error;
-  if (!formDefinition) return <Loading />;
+  if (!formIdentifier) return <Loading />;
 
   return (
     <>
-      <PageTitle title={`Edit Form: ${formDefinition?.title}`} />
-      {formDefinition && (
-        <FormEditor
-          definition={formDefinition?.rawDefinition}
-          onSave={(values) => {
-            updateFormDefinition({
-              variables: {
-                id: formId,
-                input: { definition: JSON.stringify(values) },
-              },
-            });
-          }}
-          saveLoading={loading}
-        />
-      )}
+      <PageTitle title={`Edit Form: ${formIdentifier.displayVersion.title}`} />
+      <FormEditor
+        definition={formIdentifier.displayVersion.rawDefinition}
```

Comment:
Just noting that we'll need to come back to this with https://github.com/open-path/Green-River/issues/5858 to implement the version behavior